roles/node/templates/testnet/config.toml.j2

```diff
@@ -0,0 +1,125 @@
+wasm_dir = "{{ node_wasm_dir }}"
+
+[ledger]
+genesis_time = "2024-12-05T23:07:03.000000000+00:00"
+chain_id = "housefire-alpaca.cc0d3e0c033be"
+
+[ledger.shell]
+base_dir = "{{ node_network_dir }}"
+storage_read_past_height_limit = 3600
+db_dir = "{{ node_data_dir }}"
+cometbft_dir = "{{ node_cometbft_dir }}"
+tendermint_mode = "Full"
+
+[ledger.cometbft]
+proxy_app = "tcp://127.0.0.1:{{ namada[type].ports.proxy_port }}"
+moniker = "{{ org }}"
+fast_sync = true
+db_backend = "goleveldb"
+db_dir = "data"
+log_level = "info"
+log_format = "plain"
+genesis_file = "config/genesis.json"
+priv_validator_key_file = "config/priv_validator_key.json"
+priv_validator_state_file = "data/priv_validator_state.json"
+priv_validator_laddr = ""
+node_key_file = "config/node_key.json"
+abci = "socket"
+filter_peers = false
+
+[ledger.cometbft.rpc]
+laddr = "tcp://127.0.0.1:{{ namada[type].ports.rpc_port }}"
+cors_allowed_origins = []
+cors_allowed_methods = ["HEAD", "GET", "POST"]
+cors_allowed_headers = ["Origin", "Accept", "Content-Type", "X-Requested-With", "X-Server-Time"]
+grpc_laddr = ""
+grpc_max_open_connections = 900
+unsafe = false
+max_open_connections = 900
+max_subscription_clients = 100
+max_subscriptions_per_client = 5
+timeout_broadcast_tx_commit = "10000ms"
+max_body_bytes = 1000000
+max_header_bytes = 1048576
+tls_cert_file = ""
+tls_key_file = ""
+pprof_laddr = ""
+
+[ledger.cometbft.p2p]
+laddr = "tcp://0.0.0.0:{{ namada[type].ports.p2p_port }}"
+external_address = ""
+seeds = ""
+persistent_peers = "tcp://0ed550f6f3acc48505fb194ad7966d923db5f7bb@167.99.185.160:26656,tcp://3e254524b630ee5a1d0df6aa40b4ed1370b5e51a@74.50.93.254:26656"
+upnp = false
+addr_book_file = "config/addrbook.json"
+addr_book_strict = true
+max_num_inbound_peers = 40
+max_num_outbound_peers = 10
+unconditional_peer_ids = ""
+persistent_peers_max_dial_period = "0ms"
+flush_throttle_timeout = "100ms"
+max_packet_msg_payload_size = 1024
+send_rate = 5120000
+recv_rate = 5120000
+pex = true
+seed_mode = false
+private_peer_ids = ""
+allow_duplicate_ip = false
+handshake_timeout = "20000ms"
+dial_timeout = "3000ms"
+
+[ledger.cometbft.mempool]
+recheck = true
+broadcast = true
+wal_dir = ""
+size = 5000
+max_txs_bytes = 1073741824
+cache_size = 10000
+keep-invalid-txs-in-cache = false
+max_tx_bytes = 1048576
+max_batch_bytes = 0
+
+[ledger.cometbft.consensus]
+wal_file = "data/cs.wal/wal"
+timeout_propose = "3000ms"
+timeout_propose_delta = "500ms"
+timeout_prevote = "1000ms"
+timeout_prevote_delta = "500ms"
+timeout_precommit = "1000ms"
+timeout_precommit_delta = "500ms"
+timeout_commit = "8000ms"
+double_sign_check_height = 0
+skip_timeout_commit = false
+create_empty_blocks = true
+create_empty_blocks_interval = "0ms"
+peer_gossip_sleep_duration = "100ms"
+peer_query_maj23_sleep_duration = "2000ms"
+
+[ledger.cometbft.storage]
+discard_abci_responses = false
+
+[ledger.cometbft.tx_index]
+indexer = "null"
```

Comment:
can we create space otpimization setting variable like we create in cosmos and use it here